### PR TITLE
chore: remove discussion prompt from summary

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/9710

Demo: https://www.loom.com/share/728f893c2e054fbbad82f849e89ffeaa

The AI Summary for the reflection group is no longer being used but I don't want to remove it now as it looks like we're migrating that table from RethinkDB to PG. I've created an issue: https://github.com/ParabolInc/parabol/issues/9712

### To test

- [ ] Add comments to a discussion thread 
- [ ] Go to the meeting summary and see that there's an AI Summary of the discussion thread and no discussion prompt

### CODE REVIEW

File: packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummary.tsx [hunk 1]
```diff
@@ -49,14 +41,12 @@ const WholeMeetingSummary = (props: Props) => {
...
-    const hasTopicSummary = reflectionGroups!.some((group) => group.summary)
-    const hasDiscussionSummary = !!stages?.some((stage) => stage.discussion?.summary)
-    const hasOpenAISummary = hasTopicSummary || hasDiscussionSummary
-    if (!hasOpenAISummary) return null
-    if (hasOpenAISummary && !wholeMeetingSummary) return <WholeMeetingSummaryLoading />
+    const {summary: wholeMeetingSummary, reflectionGroups, organization} = meeting
+    const hasMoreThanOneReflection =
+      (reflectionGroups?.length && reflectionGroups.length > 1) ||
+      reflectionGroups?.some((group) => group.reflections.length > 1)
+    if (!hasMoreThanOneReflection || organization.featureFlags.noAISummary) return null
```
Comment (nickoferrall) [inline packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummary.tsx:48]: If there's more than one reflection, we'll always create the meeting summary unless `noAISummary` is true, so we don't need to check if there's a discussion summary or reflection group summary (which is now removed anyway)

File: packages/client/mutations/EndRetrospectiveMutation.ts [hunk 2]
```diff
@@ -122,12 +126,9 @@ export const endRetrospectiveTeamOnNext: OnNextHandler<
...
       popEndMeetingToast(atmosphere, meetingId)
     } else {
-      const discussPhase = phases.find((phase) => phase.phaseType === 'discuss')
-      const {stages} = discussPhase ?? {}
-      const hasTopicSummary = reflectionGroups.some((group) => group.summary)
       const reflections = reflectionGroups.flatMap((group) => group.reflections) // reflectionCount hasn't been calculated yet so check reflections length
-      const hasDiscussionSummary = !!stages?.some((stage) => stage.discussion?.summary)
-      const hasOpenAISummary = hasTopicSummary || hasDiscussionSummary
+      const hasMoreThanOneReflection = reflections.length > 1
+      const hasOpenAISummary = hasMoreThanOneReflection || !organization.featureFlags.noAISummary
```
Comment (Dschoordsch) [inline packages/client/mutations/EndRetrospectiveMutation.ts:131]: -1 this should be `&&`, shouldn't it?

File: packages/server/graphql/mutations/helpers/generateDiscussionPrompt.ts [hunk 3]
```diff
@@ -40,30 +40,21 @@ const generateGroupSummaries = async (
         ({reflectionGroupId}) => reflectionGroupId === group.id
       )
       if (reflectionsByGroupId.length <= 1) return
-      const reflectionTextByGroupId = reflectionsByGroupId.map(
-        ({plaintextContent}) => plaintextContent
-      )
-      const [fullSummary, fullQuestion] = await Promise.all([
-        manager.getSummary(reflectionTextByGroupId),
+      const [fullQuestion] = await Promise.all([
```
Comment (Dschoordsch) [inline packages/server/graphql/mutations/helpers/generateDiscussionPrompt.ts:43]: -1 No need to wrap a single call in `Promise.all([...])`

**Review by Dschoordsch — CHANGES_REQUESTED** [review]
Did not test yet

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**